TreeTracker/UI/Views/AddTree/AddTreeViewController.swift

```diff
@@ -59,8 +59,8 @@ private extension AddTreeViewController {
     @IBAction func takePhotoButtonPressed() {
         let imagePicker = UIImagePickerController()
         imagePicker.delegate = self
-        imagePicker.sourceType = .camera
-        imagePicker.cameraDevice = .rear
+//        imagePicker.sourceType = .camera
```

Comment:
Get why you did this for dev purposes (the main branch now has a photo library option for debug builds to resolve this) but they need to be uncommented before merging.

Reply:
my  fault, that was for testing the simulator